lte/protos/session_manager.proto

```diff
@@ -28,6 +28,8 @@ message RuleRecord {
   uint64 bytes_rx = 4;
   string ue_ipv4 = 5;
   string ue_ipv6 = 6;
+  uint64 dropped_tx = 7;
+  uint64 dropped_rx = 8;
```

Comment:
Just to clarify: what about when the flows are removed. If sessiond send a flow removal, but the subscriber keeps sending packets, will those be reported too as part of any rule or part of "" rule as a dropped traffic?

Reply:
If a rule is removed packets will match other rules if present, or the default drop.

Reply:
the dropped stats will also increment on the flows that have DENY flow action

Reply:
Note: one rule can have both flow PERMIT/DENY matches -> so it can have both bytes_rx/tx + dropped_rx/tx

Reply:
Ok, so in case sessiond sends remove all rules/flows for a session, after pipelined have removed all the rules/flows for this subscriber, will pipelined still send reports for any traffic dropped the subscriber sends?

Reply:
discussed offline, will ignore the default rule for sessiond ue termination logic